### PR TITLE
Update UI test screenshots for OneClick after new major Matomo version was released

### PR DESCRIPTION
**Pull request by michalkleiner**
### Description:

As per the title. It's possible the tests could be tweaked to instead alter the content to not depend on an exact version. I looked into that very briefly but updating the screenshots seemed faster at this point, more so when another major Matomo version may be in distant future.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: tests/UI/expected-screenshots/OneClickUpdate_update_screen.png
Comment (sgiehl) [inline tests/UI/expected-screenshots/OneClickUpdate_update_screen.png:1]: This seems to be a regression. The Fixture should make the ExampleTracker plugin as incompatible on purpose. So we need to check why that doesn't work anymore.

Comment (michalkleiner) [reply tests/UI/expected-screenshots/OneClickUpdate_update_screen.png:1]: The update fail screenshots change seems legit though, the notification styling changed, so to me it felt like it didn't pick up those changes previously for some weird reason.

Comment (sgiehl) [reply tests/UI/expected-screenshots/OneClickUpdate_update_screen.png:1]: Actually no. The test is performing an update from the latest stable release. So before we released Matomo 5, the update was performed from 4.16.0, where the notifications still had an old styling.

Comment (michalkleiner) [reply tests/UI/expected-screenshots/OneClickUpdate_update_screen.png:1]: But at the point of running the update, is it using 4.x code or 5.x code?
Either way, I can look further into this if you give me a bit of a direction on what the correct outcome should be :)

Comment (sgiehl) [reply tests/UI/expected-screenshots/OneClickUpdate_update_screen.png:1]: The test sets up an instance for the latest stable release. So the first screens are always taken with old code (which explains the old notifications styling). The success screen will be displayed with latest code always as it's shown after a page reload.

**Review by sgiehl — APPROVED** [review]

**Pull request merged**